src/lib.rs

```diff
@@ -94,6 +94,9 @@
 //! - **`zerocopy`** — Adds support for the [`zerocopy`] crate by implementing [`AsBytes`] and
 //!   [`FromBytes`] traits for both [`f16`] and [`bf16`].
 //!
+//! - **`rand_distr`** — Adds support for the [`rand_distr`] crate by implementing [`Distribution`]
```

Comment:
CI found error in this doc, can't find `Distrubution` trait, I think you mean `Uniform` and `Normal`?

Reply:
Should be fixed now (verified with running cargo doc locally)